notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>is there a specific reason you need to use timestamp? If it's only for uniqueness consider using the UUID library instead https://docs.python.org/3/library/uuid.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>It's only for uniqueness but this content is from the template. I will discuss it with Brad later on.</p>